datafusion/sql/src/select.rs

```diff
@@ -202,7 +202,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
 
         let plan = if let Some(having_expr_post_aggr) = having_expr_post_aggr {
             LogicalPlanBuilder::from(plan)
-                .filter(having_expr_post_aggr)?
+                .filter(having_expr_post_aggr, 20)?
```

Comment:
Probably reading from SQL context is the right thing to do here? i.e. a variable?